File: tmp-mei/fgen4255027576711118260/base/Actions_0.java
package base;
public interface Actions_0{
Actions_0 $self = new Actions_0Impl();
base.Action_1 info$imm(base.Info_0 info_m$);

base.Action_1 ok$imm(Object x_m$);

base.Action_1 lazy$imm(base.MF_1 f_m$);
static base.Action_1 lazy$imm$fun(base.MF_1 f_m$, base.Actions_0 $this) {
  return ((base.Action_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Action_1) null;
}})
);
}

static base.Action_1 ok$imm$fun(Object x_m$, base.Actions_0 $this) {
  return ((base.Action_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Action_1) null;
}})
);
}

static base.Action_1 info$imm$fun(base.Info_0 info_m$, base.Actions_0 $this) {
  return ((base.Action_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Action_1) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/Actions_0.js
import * as rt from "../rt/main.js";
export class base$$Actions_0 {
  static $self = new base$$Actions_0();

  info$imm(info_m$) {
  return base$$Actions_0.info$imm$fun(info_m$, this);
}


ok$imm(x_m$) {
  return base$$Actions_0.ok$imm$fun(x_m$, this);
}


lazy$imm(f_m$) {
  return base$$Actions_0.lazy$imm$fun(f_m$, this);
}

  static lazy$imm$fun(f_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static ok$imm$fun(x_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static info$imm$fun(info_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
